### ISSUE TITLE
How to use capture with async/await?

### ISSUE THREAD

**Issue by am** [patrick-steele-idem/child-process-promise#39]
Hi, I'm trying to capture the `stdout` of a `spawn` process that uses await, here is an example:

```js
const { childProcess } = await spawn('yarn', ['install'], { capture: ['stdout'] });
console.log(childProcess.stdout.toString()); // returns [object Object]
```

I'm would like to log the complete stdout of the process once this completes. Previously used `{stdio: 'inherit'}` but since there are concurrent process it's not possible to differentiate the different processes `stdout`.

Any help is much appreciated, many thanks!

**Comment by Karolusrex** [comment]
Yes, that's a problem. I'd suggest using https://github.com/nkashyap/child-process-es6-promise instead
